superset-frontend/src/explore/components/controls/BoundsControl.jsx

```diff
@@ -87,8 +88,8 @@ export default class BoundsControl extends React.Component {
       <div>
         <ControlHeader {...this.props} />
         <FormGroup bsSize="small">
-          <Row>
-            <Col xs={6}>
+          <Row gutter={16}>
```

Comment:
I think we could use withTheme for these to get a hold of gridUnit, but it's not a blocker for this work I suppose.